src/components/PopupWithForm.js

```diff
@@ -14,26 +24,42 @@ class PopupWithForm extends Popup {
     inputs.forEach((input) => {
       values[input.name] = input.value;
     });
+
     return values;
   }
 
   _submitForm = () => {
     const inputValues = this._getInputValues();
-    this._handleFormSubmit(inputValues);
+    this._loadingState = true;
+    this.showLoading(true);
+
+    setTimeout(() => {
```

Comment:
Please remove this timeout, you should hide the loading in index.js after successful api response